libparsec/crates/protocol/schema/authenticated_cmds/shamir_create_shared_recovery_device.json5

```
@@ -0,0 +1,71 @@
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_setup",
+            "fields": [
+                {
+                    "name": "setup",
+                    // Set to `None` to clear previous Shamir recovery setup
+                    "type": "RequiredOption<ShamirRecoverySetup>"
+                }
+            ]
+        },
+        "reps": [
+            {
+                "status": "ok"
+            },
+            {
+                // Certificate is not signed by the authenticated user, or timestamp is invalid
+                "status": "invalid_certification"
```

```suggestion
                "status": "invalid_certificate"
```

For consistency: https://github.com/search?q=repo%3AScille%2Fparsec-cloud+invalid_certificate+path%3A*.json5&type=code